src/assets/server/rest/v1/docs/e-mobility-oas.json

```diff
@@ -277,32 +292,35 @@
         "description": "Register",
         "requestBody": {
           "content": {
-            "application/x-www-form-urlencoded": {
```

Comment:
> I put the type to 'x-www-form-urlencoded' because it was more user friendly in Swagger to provide data.

If the data input expect JSON format, it's not a correct change: the API documentation must give the exact data type and format expected on an endpoint, not be user friendly. It's main target is developers and the data expected must the same as the one they have to implement.

Reply:
Yes you're right, so let's stick to the standard then.
@Ninjeneer Replace all application/x-www-form-urlencoded by application/json.
Thanks!

Reply:
Done